apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/parser/introspection/IntrospectionSchema.kt

```diff
@@ -162,19 +162,19 @@ data class IntrospectionSchema(
 
     fun IntrospectionQuery.Schema.toIntrospectionSchema(): IntrospectionSchema {
       return IntrospectionSchema(
-          queryType = queryType?.name ?: "query",
-          mutationType = mutationType?.name ?: "mutation",
-          subscriptionType = subscriptionType?.name ?: "subscription",
+          queryType = queryType?.name ?: "Query",
```

Comment:
Just to double check Is `Query` supposed to be uppercase?

Reply:
That's my understanding: 

```
While any type can be the root operation type for a GraphQL operation, the type system definition language can omit the schema definition when the query, mutation, and subscription root types are named Query, Mutation, and Subscription respectively.
```

I do expect this to never happen in practice though. I don't see how an introspection result could not have a "query" root type since it was used to get the introspection result. I took the conservative option but maybe we should throw here